### ISSUE TITLE
strange label display issue (how many labels can I display? )

### ISSUE THREAD

**Issue by xarthurx** [libigl/libigl#1195]
I'm trying to display labels in the viewer, however, strange behaviour happens when the # of labels increases.

pic 1: smaller # of labels (~1000) **NO PROBLEM**:
![image](https://user-images.githubusercontent.com/1921878/57691120-99661000-7643-11e9-822e-efdc51057d82.png)

pic 2 - 4: large # of labels (>6000) **PROBLEM"":

2:
![image](https://user-images.githubusercontent.com/1921878/57691204-d16d5300-7643-11e9-8d2b-0b36026026fe.png)

3.
![image](https://user-images.githubusercontent.com/1921878/57691234-efd34e80-7643-11e9-84e3-6252be657eb4.png)

4.
![image](https://user-images.githubusercontent.com/1921878/57691244-f792f300-7643-11e9-9b5b-ad597774b0d8.png)

for pic 2-4, as long as I move some labels out of the viewer (by zooming in / move the obj), the labels that are missing in pic2 start to appear little by little.

Is there a specific # of labels that the viewer can display?
What is the reason behind the issue?

#### Check all that apply (change to `[x]`)
- [x] Windows
- [ ] Mac OS X
- [ ] Linux
- [x] I have tried the `dev` branch and the problem persists

See https://libigl.github.io/CONTRIBUTING/#bugreport for more tips.

**Comment by jdumas** [comment]
Yes it's a known issue. See #719 and #1109 for related discussions.

**Comment by xarthurx** [comment]
Thanks for the info.
It seems already been a year since https://github.com/libigl/libigl/issues/719.
Is the issue of pure ImGui or sth related with the integration? (Seems the latter one from the discussion in the above issue)

**Comment by jdumas** [comment]
It's a bit of both, but there are multiple issues with the current support for labels, as discussed in #1109. It's been a year but it's also not a priority so we haven't gotten the time to work on this.
